dashboards/victoriametrics-cluster.json

```diff
@@ -4124,6 +4124,111 @@
           ],
           "title": "Rows ignored for last 1h ($instance)",
           "type": "timeseries"
+        },
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "$ds"
+          },
+          "description": "Shows the restarts of the cluster components",
```

Comment:
I'd suggest to add more details to the description:
1. What exactly it shows (amount of restarts per job)
2. What it means for a user
3. What user can do 

Please note, dashboard is a troubleshooting tool in the first place. Panels should be self-descriptive and helpful for everyone.

I'd also recommend removing relation to `cluster components` to make this panel more generic. Because the second thing I'd like to ask is to port this panel to the other dashboards. In the same way as we have panels CPU or Mem usage ported across other dashboards.

Reply:
Done.